vividus-to-xray-exporter/src/main/resources/org/vividus/jira/spring.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+       http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd"
+       default-lazy-init="true">
+
+    <bean id="caching-credentials" class="org.vividus.http.context.CachingCredentialsHttpContextFactory" >
+        <constructor-arg index="0" value="${jira.username}"/>
+        <constructor-arg index="1" value="${jira.password}"/>
+        <constructor-arg index="2" value="${jira.endpoint}"/>
+    </bean>
+
+    <bean id="noop" class="org.vividus.http.context.NoopHttpContextFactory" />
+
+    <bean id="jiraFacade" class="org.vividus.jira.JiraFacade" />
+
+    <bean id="jiraClient" class="org.vividus.jira.JiraClient">
+        <constructor-arg index="0" value="${jira.endpoint}" />
+        <constructor-arg index="1">
+            <bean class="org.vividus.http.client.HttpClient" factory-bean="httpClientFactory"
+                  factory-method="buildHttpClient">
+                <constructor-arg>
+                    <bean parent="httpClientConfig">
+                        <property name="headersMap">
+                            <bean factory-bean="propertyParser" factory-method="getPropertyValuesByPrefix">
+                                <constructor-arg value="jira.http.header." />
+                            </bean>
+                        </property>
+                    </bean>
+                </constructor-arg>
+            </bean>
+        </constructor-arg>
+        <constructor-arg index="2" ref="${jira.http-context-type}" />
```

Comment:
I'm sorry for confusion, let's rename the property one more time: `jira.http.context-type` (it will bring consistency with other `jira.http.*` properties)